[2022-10-24T22:45:14.987+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.truncate_database manual__2022-10-24T22:35:47.983631+00:00 [queued]>
[2022-10-24T22:45:15.000+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: sales_datawarehouse.truncate_database manual__2022-10-24T22:35:47.983631+00:00 [queued]>
[2022-10-24T22:45:15.000+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T22:45:15.001+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 3
[2022-10-24T22:45:15.002+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-10-24T22:45:15.018+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): truncate_database> on 2022-10-24 22:35:47.983631+00:00
[2022-10-24T22:45:15.022+0000] {standard_task_runner.py:54} INFO - Started process 26338 to run task
[2022-10-24T22:45:15.025+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'sales_datawarehouse', 'truncate_database', 'manual__2022-10-24T22:35:47.983631+00:00', '--job-id', '509', '--raw', '--subdir', 'DAGS_FOLDER/sales_dw.py', '--cfg-path', '/tmp/tmpt2t8kaas']
[2022-10-24T22:45:15.027+0000] {standard_task_runner.py:83} INFO - Job 509: Subtask truncate_database
[2022-10-24T22:45:15.028+0000] {dagbag.py:525} INFO - Filling up the DagBag from /opt/***/dags/sales_dw.py
[2022-10-24T22:45:15.586+0000] {task_command.py:384} INFO - Running <TaskInstance: sales_datawarehouse.truncate_database manual__2022-10-24T22:35:47.983631+00:00 [running]> on host 83cd0ab98720
[2022-10-24T22:45:15.660+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pedromenegat
AIRFLOW_CTX_DAG_ID=sales_datawarehouse
AIRFLOW_CTX_TASK_ID=truncate_database
AIRFLOW_CTX_EXECUTION_DATE=2022-10-24T22:35:47.983631+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-10-24T22:35:47.983631+00:00
[2022-10-24T22:45:15.671+0000] {base.py:71} INFO - Using connection ID '***_localhost' for task execution.
[2022-10-24T22:45:15.731+0000] {sql.py:315} INFO - Running statement: 
            TRUNCATE TABLE funcionarios, categorias, vendas;
            , parameters: None
[2022-10-24T22:45:15.763+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=sales_datawarehouse, task_id=truncate_database, execution_date=20221024T223547, start_date=20221024T224514, end_date=20221024T224515
[2022-10-24T22:45:15.800+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2022-10-24T22:45:15.842+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
